ui/src/routes/devices.$id.settings.access._index.tsx

```diff
@@ -148,12 +152,13 @@ export default function SettingsAccessIndexRoute() {
 
   // Function to update TLS state - accepts a mode parameter
   const updateTlsState = useCallback(
-    (mode: string, cert?: string, key?: string) => {
+    (mode: string, enforce: boolean, cert?: string, key?: string) => {
       const state = { mode } as TLSState;
       if (cert && key) {
         state.certificate = cert;
         state.privateKey = key;
       }
+      state.enforce = enforce;
```

Comment:
Shouldn't be able to set enforce to `true` if the `mode` is `"disabled"`, so we need to make sure that is enforced here.

Reply:
Resolved by checking "disabled" and forcing false if so.